listmonk/templates/ingress.yaml

```diff
@@ -2,11 +2,8 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
```

Comment:
This is not going to work, should use HTTPRoute for the GKE Gateway API.

https://cloud.google.com/kubernetes-engine/docs/concepts/gateway-api

Reply:
fixed!